feature-libs/product-configurator/rulebased/components/overview-menu/configurator-overview-menu.component.html

```diff
@@ -17,31 +17,34 @@
   let-level="level"
   let-idPrefix="idPrefix"
 >
-  <ng-container *ngFor="let group of overviewGroups">
-    <div [ngClass]="getGroupLevelStyleClasses(level)">
-      <button
-        class="cx-menu-item"
-        role="tab"
-        [attr.aria-label]="
-          'configurator.a11y.groupName'
-            | cxTranslate: { group: group.groupDescription }
-        "
-        (click)="navigateToGroup(idPrefix, group.id)"
-      >
-        <span aria-hidden="true"> {{ group.groupDescription }}</span>
-      </button>
-      <ng-container *ngIf="group.subGroups?.length > 0">
-        <ng-container
-          *ngTemplateOutlet="
-            groups;
-            context: {
-              overviewGroups: group.subGroups,
-              level: level + 1,
-              idPrefix: idPrefix + group.id
-            }
+  <ul>
```

Comment:
This breaking change needs to be documented for customers

Reply:
Done!